File: Maximum_Number_of_Coins_You_Can_Get.js
/**
 * @param {number[]} piles
 * @return {number}
 */
var maxCoins = function(piles) {
    let i=0;
      let j=piles.length-1;
      let sum =0;
      piles.sort((a,b)=>{return a-b    })
       while(i<j){
        console.log(piles[i],piles[j])
        sum+=piles[j-1]
          j-=2;
          i++;
      }
      
      return sum
  };

File: main-One/maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:

        #  1 2 2 4 7 8
        piles = sorted(piles)

        j= len(piles)-2
        i = 0
        sum_ =0

        while i<j:
            sum_+=piles[j]
            j-=2
            i+=1
        

        return sum_




        